test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -449,3 +449,32 @@
       initNodes: 5000
       initPods: 2000
       measurePods: 5000
+
+- name: PendingPodsWithMixedChurn
```

Comment:
I have another idea, which perhaps is more realistic and something we would like tested. 

Create pods first (they are unschedulable). Then create Nodes (perhaps have a ticker for it). We would have to combine that with pending pods that are unschedulable for different reasons (like waiting for pod affinity).

@adtac didn't we have a test similar to what I'm describing?

Reply:
Another very important one:
New nodes show up, but with not-ready taint, then the taint is removed.

Reply:
> Create pods first (they are unschedulable). Then create Nodes (perhaps have a ticker for it).

SG. However, in that case we need to enable both `SkipWaitToCompletion` and `collectMetrics`, which is not supported yet:

https://github.com/kubernetes/kubernetes/blob/b5808c6df97a786047c04a35371b20104a1dd28d/test/integration/scheduler_perf/scheduler_perf_test.go#L379-L381

Reply:
>didn't we have a test similar to what I'm describing?

no, I don't think we did. multiple createNodes in one workload was one of my motivations for the scheduler_perf rewrite, but I never got around to opening a PR with such a workloda even though I tested it locally :/

>However, in that case we need to enable both SkipWaitToCompletion and collectMetrics

True. Unfortunately, adding support for specifying both parameters seems to be quite difficult because of how metrics/throughput collection works.

The node taint one is a good idea :+1:  but, of course, it requires a modifyNodes op or something like that.